linkcheck/htmlutil/linkparse.py

```diff
@@ -175,6 +175,10 @@ def start_element (self, tag, attrs, element_text, lineno, column):
         log.debug(LOG_CHECK, "line %d col %d", lineno, column)
         if tag == "base" and not self.base_ref:
             self.base_ref = attrs.get("href", u'')
+        if tag =="a" and attrs.get_true('class', u''):
+           if ("broken_link" in attrs.get('class')):
```

Comment:
I wonder if this should be configurable?  Some sites might use a different class name.

I think we should `.split()` the class attribute so substring matching wouldn't cause accidental matches for unrelated classes like `<a class="unbroken_linkage">` or something.